packages/server/src/api/controllers/table/external.ts

```diff
@@ -98,7 +98,7 @@ export async function bulkImport(
     table = processed.table
   }
 
-  await handleRequest(Operation.BULK_CREATE, table._id!, {
+  await handleRequest(Operation.BULK_UPSERT, table._id!, {
```

Comment:
is there any use in the `BULK_CREATE` operation anymore for SQL? Should we just have both of those operations go through the bulk upsert process?

Reply:
It appears to still be used in the Google Sheets integration.

Reply:
Oh, in SQL. I'm not quite sure what you mean... What do you mean by "both of those"?